src/word_wrap/WrapperTest.java

```diff
@@ -14,7 +14,7 @@ public void emptyString() {
 
     @Test
     public void wordShorterThanWrapColumn() {
-        assertEquals("word", Wrapper.wrap("word", 7));
```

Comment:
These test changes felt very hacky, but also meant that the next refactor was much simpler to see.